File: API/Models/Rental.cs
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

namespace RentMovie.Models
{
    public class Rental
    {
        [Key]
        public int RentalId { get; set; }
        [ForeignKey("UserId")]
        public int UserId { get; set; }
        [ForeignKey("MovieId")]
        public int MovieId { get; set; }
        public DateTime RentalDate { get; set; }
        public DateTime DueDate { get; set; }
        public string IsReturned { get; set; }

      /*  [ForeignKey("UserId")]
        public User User { get; set; }

        [ForeignKey("MovieId")]
        public Movie Movie { get; set; }
      */
    }
}

File: UI/app/model/Rental.js
Ext.define('RentMovieApp.model.Rental', {
    extend: 'Ext.data.Model',

    fields: [
        { name: 'rentalId', type: 'int' },
        { name: 'userId', type: 'int' },
        { name: 'movieId', type: 'int' },
        { name: 'rentalDate', type: 'auto' },
        { name: 'dueDate', type: 'auto' },
        { name: 'isReturned', type: 'string' },
    ]
});